src/tuwien/auto/calimero/dptxlator/DPTXlator4ByteSigned.java

```diff
@@ -120,6 +120,20 @@ public class DPTXlator4ByteSigned extends DPTXlator
 	 */
 	public static final DPT DPT_DELTA_TIME = new DPT("13.100", "time lag", "-2147483648", "2147483647", "s");
 
+	// metering
+
+	/**
+	 * DPT ID 13.1200, volume liquid [l] for water/heat meter total consumption; values from <b>-2147483648</b> to
+	 * <b>2147483647</b> liter.
+	 */
+	public static final DPT DPT_DELTA_VOLUME_LIQUID_LITRE = new DPT("13.1200", "volume liquid [l]", "-2147483648", "2147483647", "l");
```

Comment:
LITRE -> LITER

Reply:
this is actually how KNX.org defined it.... https://www.knx.org/wAssets/docs/downloads/Certification/Interworking-Datapoint-types/03_07_02-Datapoint-Types-v02.02.01-AS.pdf
I can change if you want, for me it sounds unfamiliar as well....

Reply:
They just used BE, not AE (english).

Reply:
Shall I change it as suggested?

Reply:
Yes, please (liter is used everywhere else in the source code)

Reply:
done